{
  "comments": [
    {
      "key": {
        "uuid": "224c6d49_4c6472d9",
        "filename": "src/SpecialMathWikibase.php",
        "patchSetId": 2
      },
      "lineNbr": 128,
      "author": {
        "id": 787
      },
      "writtenOn": "2020-09-10T16:51:01Z",
      "side": 1,
      "message": "This error message does not sound right to me? Shouldn\u0027t be \"An unknown error occurred while fetching data from Wikibase.\" ?",
      "range": {
        "startLine": 128,
        "startChar": 25,
        "endLine": 128,
        "endChar": 85
      },
      "revId": "b92a11d6094a225708140d608c413d2b7f4954c9",
      "serverId": "e9e9afe9-4712-486d-8885-f54b72dd1951",
      "unresolved": true
    }
  ]
}